### PR TITLE
refactor: make dropdowns uniform

### LINKED ISSUE #4586: Always right-align the actions dropdown menu
Many of our dropdown menus look quite strange on large screens because they are not right-aligned.  Here is an example from the stock assignment registry:

image

We should make it an effort to always right align our menus in the future.  To close this issue, someone should go through and make sure that we are consistently right-aligning the actions in all registries.

### LINKED ISSUE #4587: Stock Assignment Registry needs dropdown-header class on dropdown header
In the dropdown menu of the stock assignment registry, we need to put the `.dropdown-header` class on the dropdown header.  Currently acts like a clickable link.  See below:

image

### PR DESCRIPTION
**Pull request by jniles**
Goes through all dropdowns in the application and makes them uniform by aligning them to the right (`.text-right`) and also making sure they don't overflow but instead use `text-ellipsis` to overflow.  I've also put headers on the dropdown menus that do not have headers and expanded the text of those where it would be more useful.

Closes #4586.
Closes https://github.com/IMA-WorldHealth/bhima/issues/4587

### CODE REVIEW

File: client/src/modules/account_reference/templates/action.cell.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </a>
 
   <ul data-row-menu="{{row.entity.abbr}}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
-    <li class="dropdown-header text-ellipsis" style="max-width:250px;">{{row.entity.abbr}}</li>
+    <li class="bh-dropdown-header">{{row.entity.abbr}}</li>
```
Comment (mbayopanda) [inline client/src/modules/account_reference/templates/action.cell.html:9]: 👍

File: client/src/modules/dashboards/indicators_files_registry/indicators_files_registry.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function IndicatorsFilesRegistryController(
 ) {
   const vm = this;
 
-  const cacheKey = 'AdmissionRegistry';
+  const cacheKey = 'IndicatorFilesRegistry';
```
Comment (mbayopanda) [inline client/src/modules/dashboards/indicators_files_registry/indicators_files_registry.js:16]: 👍

File: client/src/modules/employees/templates/action.cell.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
   </a>
 
   <ul data-row-menu="{{ row.entity.reference }}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
-    <li class="dropdown-header">{{row.entity.reference}}</li>
+    <li class="bh-0dropdown-header">{{row.entity.reference}}</li>
```
Comment (mbayopanda) [inline client/src/modules/employees/templates/action.cell.html]: `bh-dropdown-header` i think

**Review by mbayopanda — COMMENTED** [review]
There are a lot of file changed 😄 

The use of ellipsis is very nice, i just discover for some registries there also divider under the `bh-dropdown-header` for example : 
- Administration -> Configuration des outils d'analyse
- Administration -> Reference de compte
- Administration -> Reference Seuil de rentabilite
- Collecte de données -> Kit de collecte de données ->  Configuration des formulaires
- Collecte de données -> Kit de collecte de données -> Gestion des formulaires
- Collecte de données -> Kit de collecte de données -> Gestion des listes de choix

For the account module, the ellipsis doesn't work we have something like : 
image

Instead of having something like : 
image

- Registre des employes there is a `bh-0dropdown-header` instead of `bh-dropdown-header`

**Comment by jniles** [comment]
@mbayopanda changes landed.

File: client/src/less/bhima-bootstrap.less [hunk 4]
```diff
@@ -539,3 +539,16 @@ div.ui-grid-cell .form-group.has-error input.ng-invalid {
...
+
+/* extends .dropdown-header with .text-ellipsis and max-width */
+.bh-dropdown-header {
+	color: #777;
+	display: block;
+	font-size: 12px;
+	line-height: 1.42857143;
+  max-width:250px;
+	overflow: hidden;
+	padding: 3px 20px;
```
Comment (mbayopanda) [inline client/src/less/bhima-bootstrap.less:551]: 👍

**Review by mbayopanda — APPROVED** [review]
bors r+

**Pull request merged**